client/components/activity-card-list/index.jsx

```diff
@@ -209,6 +209,10 @@ class ActivityCardList extends Component {
 		const pageLogs = this.splitLogsByDate( visibleLogs.slice( ( actualPage - 1 ) * pageSize ) );
 		const showLimitUpsell = visibleLogs.length < logs.length && actualPage >= pageCount;
 
+		if ( pageLogs.length === 0 ) {
+			return 'Empty!';
```

Comment:
Doesn't this need to be translated? I see many other Pagination labels in this component that have not been translated, any specific reason? 🤔

Reply:
It definitely would, if we ever wanted to put this into production. Fortunately in this case, we'll eventually have an illustration (and maybe a little copy) to replace the `Empty!` string.

Reply:
As for the other labels, those need to be translated as well, but I tried to keep my first draft of changes as finely scoped as possible.

Reply:
May be we can track that somewhere so that it doesn't get unnoticed?

Reply:
That "`DO NOT MERGE` label" works fine. Thank you.